runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -560,6 +581,27 @@ void stopPlugin(Plugin plugin, PluginListItem listItem)
 		});
 	}
 
+	private List<String> getPinnedPluginNames()
+	{
+		final String config = configManager.getConfiguration(RUNELITE_GROUP_NAME, PINNED_PLUGINS_CONFIG_KEY);
+
+		if (config == null)
+		{
+			return new ArrayList<>();
+		}
+
+		return Arrays.asList(config.split(","));
```

Comment:
Use Splitter.on (and make it static final somewhere at the top) and then splitToList(0